[14.0][ADD] account_move_substate

bosd opened the pull request:
Superseeds: #1001

ps-tubtim committed:
[14.0][ADD] account_move_substate

bosd committed:
[FIX] account_move_substate: mail template

bosd committed:
[IMP] account_move_substate: Improve views

add substate to tree view, group buy and search

and fix form view

bosd committed:
[IMP] account_move_substate: add tests

[IMP] account_move_substate: add tests

bosd committed:
[IMP] account_move_substate: documentation

bosd commented:
@Dranyel-Bosd Can you functionally test this?

/ocabot rebase

bosd commented:
@Saran440 Can you please review this one?

Dranyel-Bosd reviewed (approved):
Functional review :+1:

bosd commented:
@Saran440 Can you please review?

Saran440 reviewed (commented):
Functional Test :+1: 

Please rebase commit to 1 commit.

bosd commented:
@Saran440 Done!

bosd committed:
Merge branch 'OCA:14.0' into 14.0

bosd committed:
[IMP] account_move_substate: documentation

bosd commented:
Rebased and added documentation commit (As discovered in V15/16 port)

@oca/account-invoicing maintainers
cc: @sbidoul

Can we please get a merge 🙏

bosd commented:
@dreispt Can we please get a merge?

dreispt commented:
/ocabot merge nobump
